### ISSUE TITLE
Popups for details?

### ISSUE THREAD

**Issue by ghost** [spokanelibrary/ent#53]
On the SCLD site, I like that you can click on a title and the details open in a popup window. Makes it much easier to see where items are located, read a description, and place holds without going back and forth on the list. Is this something we can do with HIP? or do you think this will cause other problems?

**Comment by jsmithley** [comment]
I like this particular enhancement, too.

**Comment by seangirard** [comment]
Yes, we can do it. That's called a modal window and you can set Enterprise to show the detail screen either in a modal window (like you are talking about) or a full page load (like it is setup now).

The reasons I went with full page are:
1) There's no good way to link to a detail screen if we go with modal view.
2) Modal view doesn't work very well on mobile.

I'm open to discussion. Those are the issues I see.
